### DISCUSSION TITLE
[Notice] Community Weekly Meeting (January 12, 2022)

### DISCUSSION THREAD

**Discussion by wang-yeliang** [apache/apisix#6032] [General]
Hi, here is the schedule for the coming Weekly Meeting.

Schedule

- Date: on Wednesday, January 12, at 20:00 ~ 22:00 (UTC +8)
- Meeting Link: [https://meeting.tencent.com/dm/VLjTH3CU3QVi](https://meeting.tencent.com/dm/VLjTH3CU3QVi)
- Content: 

1. Application scenario Share(60min): APISIX - 30min/APISIX Ingress Controller - 30min
2. Free Talk(60min)

Note

- If you have any questions, please comment below or contact [wangyeliang@apache.org](mailto:wangyeliang@apache.org)
- Kindly let me know if you need Google Meet or Zoom as communication tools.

**Comment by IT-coach-666** [comment]
I am concerning the following issue recently, could you add this issue to discuss?
https://github.com/apache/apisix/issues/6034

**Reply by wang-yeliang** [reply]
Please add your follow issue to https://docs.qq.com/doc/DUGN1clRjT0pxUk5N
